wlutil/fedora/overlay/etc/fstab

```diff
@@ -0,0 +1 @@
+UUID=12bd5ff0-295f-40fe-a8ba-d08878f38173  / ext4    defaults,noatime,x-systemd.device-timeout=300s,x-systemd.mount-timeout=300s 0 0
```

Comment:
(For the record, a patched `/etc/fstab` is necessary since the marshal images omit the boot partition.)

While the `fstab` entry for `/` is mostly irrelevant since the initramfs is responsible for mounting the rootfs in our current setup, the hardcoded UUID could become an unwelcome surprise if that ever changes in a future Fedora bump.

Unfortunately, an explicit device node won't generalize to both QEMU (`/dev/vda`) and FireSim (`/dev/iceblk`), but forunately, the base filesystem comes with a default label that can be used:
```suggestion
LABEL=_/  / ext4    defaults,noatime,x-systemd.device-timeout=300s,x-systemd.mount-timeout=300s 0 0
```